exasol_transformers_extension/resources/templates/with_spans/span_token_classification_udf.jinja.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE {{ language_alias }} SET SCRIPT "TE_TOKEN_CLASSIFICATION_UDF_WITH_SPAN"(
+    device_id INTEGER,
+    bucketfs_conn VARCHAR(2000000),
+    sub_dir VARCHAR(2000000),
+    model_name VARCHAR(2000000),
+    text_data VARCHAR(2000000),
+    docid INTEGER,
+    text_data_char_begin INTEGER,
+    text_data_char_end INTEGER,
+    aggregation_strategy VARCHAR(2000000)
+    ORDER BY {{ ordered_columns | join(" ASC,") }} ASC
+)EMITS (
+    bucketfs_conn VARCHAR(2000000),
+    sub_dir VARCHAR(2000000),
+    model_name VARCHAR(2000000),
+    text_data VARCHAR(2000000),
+    docid INTEGER,
+    text_data_char_begin INTEGER,
+    text_data_char_end INTEGER,
+    aggregation_strategy VARCHAR(2000000),
+    start_pos INTEGER,
```

Comment:
Ah, here the return interface needs to be different. Start_pos and end_pos are from the non span variant and token_* is new. Both is incorrect. We keep the interface of the non span variant, except we replace start_pos and end_pos with an entity span, so entity_docid, entity_char_begin, ...